proto/babylon/finality/v1/finality.proto

```diff
@@ -64,4 +66,7 @@ message FinalityProviderSigningInfo {
     // missed_blocks_counter defines a counter to avoid unnecessary array reads.
     // Note that `Sum(MissedBlocksBitArray)` always equals `MissedBlocksCounter`.
     int64 missed_blocks_counter = 3;
+    // Timestamp until which the validator is jailed due to liveness downtime.
+    google.protobuf.Timestamp jailed_until = 4
+    [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (amino.dont_omitempty) = true];
```

Comment:
if this cannot be null, what time it will be when the FP is not jailed?

Reply:
got it, thanks!

Reply:
What do you think about making this nullable and we use a nil check to determine the jailing status of a FP?

Reply:
Hmmm the jailing status should be obtained from the fp proto other than the signing info but it can also be inferred by checking whether the value is zero